src/RemoteSymbolProvider/MicrosoftSymbolServerSymbolProviderTest.cpp

```diff
@@ -114,8 +118,8 @@ TEST_F(MicrosoftSymbolServerSymbolProviderTest, RetrieveModuleNotFound) {
 
   orbit_base::StopSource stop_source{};
   symbol_provider_.RetrieveSymbols(module_id, stop_source.GetStopToken())
-      .Then(executor_.get(), [](ErrorMessageOr<CanceledOr<std::filesystem::path>> result) {
-        EXPECT_THAT(result, HasError("not found"));
+      .Then(executor_.get(), [](const SymbolLoadingOutcome& result) {
+        EXPECT_TRUE(orbit_symbol_provider::IsNotFound(result));
```

Comment:
pers pref:
```suggestion
        ASSERT_TRUE(orbit_symbol_provider::IsNotFound(result));
        EXPECT_EQ(orbit_symbol_provider::GetNotFoundMessage(result), "Symbols not found in Microsoft symbol server");
```

(Same for Stadia I guess)